tests/test-extensions/heisenberg-extension/src/main/java/org/mule/test/heisenberg/extension/HeisenbergOperations.java

```diff
@@ -543,6 +544,28 @@ public InputStream nameAsStream(@Config HeisenbergExtension config) {
     return new ByteArrayInputStream(sayMyName(config).getBytes());
   }
 
+  @MediaType(value = TEXT_PLAIN, strict = false)
+  public InputStream nameAsStreamConnected(@Config HeisenbergExtension config, @Connection HeisenbergConnection connection) {
```

Comment:
what's the rationale behind this? This doesn't make much sense and keeps testing the implementation instead of the behavior